app/routes/offender-manager-utilisation.js

```diff
@@ -0,0 +1,9 @@
+const getOffenderManagerUtilisationTable = require('../services/get-offender-manager-utilisation-table')
+
+module.exports = function (router) {
+  router.get('/offender-manager-utilisation/:userId/:year/', function (req, res, next) {
```

Comment:
I think this should be `caseload-utilisation` as the same view should ultimately be extended to include filters for different levels of Org unit. I.e. LDU, Team etc.

As per https://noms-wmt-ux.herokuapp.com/alpha/v3-2/capacity in the prototype

`userId` also implies it's always a user of the system, which probably isn't the case. 

What about something like `/caseload-utilisation/offendermanager/:id/:year/`

I'm thinking this would allow us to evolve the page for different levels of org unit by just changing the `offendermanager` bit for the appropriate category. What do you think?